libkbfs/folder_branch_ops.go

```diff
@@ -3081,18 +3104,19 @@ func (fbo *folderBranchOps) renameLocked(
 	}
 
 	// does name exist?
-	if de, ok := newPBlock.Children[newName]; ok {
+	replacedDe, ok := newPBlock.Children[newName]
+	if ok {
```

Comment:
`replacedDe` and `ok` are only used with in the `if` block, so it seems it's better to have the assignment within `if` line like before, so the two variables are not left around?